src/devtools/useAtomDevtools.ts

```diff
@@ -51,6 +67,18 @@ export function useAtomDevtools<Value>(
   }
 
   const [value, setValue] = useAtom(anAtom, scope)
+
+  const setAnyAtom = (value: Value) => {
```

Comment:
I'm not sure what `Any` means here. Can we name something like `setValueSafely`?